app/controllers/admins_controller.rb

```diff
@@ -1,43 +1,42 @@
 class AdminsController < ApplicationController
-    before_action :authenticate_user!
+  before_action :authenticate_user!
     
-    def index
-      @items=Item.where("status = 'pending'")
-      @procedures=Procedure.where("status = 'pending'")
-      @user=current_user
-    end
+  def index
+    @items = Item.where(status: 'pending')
+    @procedures = Procedure.where(status: 'pending')
+  end
 
-    def approve
-        @item=Item.find(params[:id])
-        @item.status="approved"
-        @item.save
-        redirect_to admins_path
-    end
+  def approve_item
+    @item = Item.find(params[:id])
+    @item.status = 'approved'
+    @item.save
+    redirect_to admins_url
+  end
     
-    def approve_p
-        @procedure=Procedure.find(params[:id])
-        @procedure.status="approved"
-        @procedure.save
-        redirect_to admins_path
-    end
+  def approve_procedure
+    @procedure = Procedure.find(params[:id])
+    @procedure.status = 'approved'
+    @procedure.save
+    redirect_to admins_url
+  end
 
-    def edit 
-        @item=Item.find(params[:id])
-    end
+  def edit 
+    @item = Item.find(params[:id])
+  end
     
-    def edit_p
-        @procedure=Item.find(params[:id])
-    end
+  def edit_p
```

Comment:
make the name more descriptive

Reply:
Deleted edit and edit_p because they are not needed.  Edit methods are routed through their respective controllers.